examples/streamlit_example.py

```diff
@@ -0,0 +1,51 @@
+"""Example script showing example dashboard hyperdrive."""
+
+# pylint: disable=pointless-statement
+import time
+
+from fixedpointmath import FixedPoint
+
+from agent0 import LocalChain, LocalHyperdrive, PolicyZoo
+
+# Initialization
+local_chain_config = LocalChain.Config()
+chain = LocalChain(local_chain_config)
+
+initial_pool_config = LocalHyperdrive.Config()
+hyperdrive0 = LocalHyperdrive(chain, initial_pool_config)
+
+agent0 = hyperdrive0.init_agent(
+    base=FixedPoint(100000),
+    eth=FixedPoint(100),
+    name="random_bot",
+    # The underlying policy to attach to this agent
+    policy=PolicyZoo.random,
+    # The configuration for the underlying policy
+    policy_config=PolicyZoo.random.Config(rng_seed=123),
+)
+
+initial_pool_config = LocalHyperdrive.Config()
+hyperdrive1 = LocalHyperdrive(chain, initial_pool_config)
+
+agent1 = hyperdrive1.init_agent(
+    base=FixedPoint(100000),
+    eth=FixedPoint(100),
+    name="random_bot",
+    # The underlying policy to attach to this agent
+    policy=PolicyZoo.random,
+    # The configuration for the underlying policy
+    policy_config=PolicyZoo.random.Config(rng_seed=345),
+)
+
+# Run the dashboard in a subprocess.
+# This command should automatically open a web browser that connects.
+chain.run_dashboard(blocking=False)
+
+# Make trades slowly
+for _ in range(100):
```

Comment:
have you tried this? 100 sec seems too fast to see what's going on; the browser crashes on `chain.cleanup()` right? May want to bump this up to 1,000 and use a `DEMO_LENGTH_IN_SECONDS` const or something at the top so the user understands why the example crashes when it does.